app/controllers/api/sessions_controller.rb

```diff
@@ -0,0 +1,34 @@
+class Api::SessionsController < Api::ApplicationController
+  def new
+  end
+
+  def create
+    user = User.find_by(email: params[:session][:email].downcase)
```

Comment:
please add a space here for better readability also before the return statements in any block in general

Reply:
between def create and user = User.find_by() ?

Reply:
sorry for the confusion, after line 6 between where you declare the user and render